### PR TITLE
[devtools-blog] Update hero image

### PR DESCRIPTION
**Pull request by jecfish**

### CODE REVIEW

File: site/en/blog/memory-inspector-extended-cpp/index.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ authors:
   - michalpitr
 description: >
     Learn how we improved debugging C/C++ WebAssembly apps with the Memory Inspector.
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/jOB4ysk52NROohGKdp85.png'
```
Comment (jecfish) [inline site/en/blog/memory-inspector-extended-cpp/index.md:8]:
```suggestion
hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/UWLghnj8M1chllDY0Iir.jpg'
alt: ''
```

File: site/en/blog/memory-inspector-extended-cpp/index.md [hunk 2]
```diff
@@ -5,6 +5,8 @@ authors:
   - michalpitr
 description: >
     Learn how we improved debugging C/C++ WebAssembly apps with the Memory Inspector.
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/UWLghnj8M1chllDY0Iir.jpg'
+alt: ''
```
Comment (philipwalton) [inline site/en/blog/memory-inspector-extended-cpp/index.md:9]: Looks like the images has text in it, so the alt value here should probably at least include that text.

**Review by philipwalton — CHANGES_REQUESTED** [review]

File: site/en/blog/memory-inspector-extended-cpp/index.md (hunk 2 above)
Comment (jecfish) [inline site/en/blog/memory-inspector-extended-cpp/index.md:9]:
```suggestion
alt: 'Learn how we improved debugging C/C++ WebAssembly apps with the Memory Inspector'
```

File: site/en/blog/memory-inspector-extended-cpp/index.md [hunk 3]
```diff
@@ -5,6 +5,8 @@ authors:
   - michalpitr
 description: >
     Learn how we improved debugging C/C++ WebAssembly apps with the Memory Inspector.
```
Comment (jecfish) [inline site/en/blog/memory-inspector-extended-cpp/index.md:7]:
```suggestion
    Learn how we improved debugging C/C++ WebAssembly apps with the Memory Inspector
```

**Review by philipwalton — APPROVED** [review]

**Pull request merged**